sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala

```diff
@@ -359,6 +359,9 @@ class DDLParserSuite extends AnalysisTest with SharedSparkSession {
       false,
       LocalTempView)
     comparePlans(parsed2, expected2)
+
+    val v3 = "CREATE TEMPORARY VIEW a.b AS SELECT 1"
+    intercept(v3, "It is not allowed to add database prefix")
```

Comment:
@cloud-fan, seems like this test is flaky:

https://github.com/apache/spark/runs/4209199644
https://github.com/apache/spark/runs/4235793982
https://github.com/apache/spark/runs/4223996038

See also SPARK-37308

Reply:
I thought this was an old test so didn't take an action but probably we should revert this one - seems pretty frequently failing.

Reply:
Im gonna partially revert this change alone for now because other tests seem passing fine.

Reply:
https://github.com/apache/spark/pull/34639